### PR TITLE
Use format player net

### LINKED ISSUE #56: Create a FormatPlayerNetID subroutine
Create a helper routine that standardizes formatting PlayerNetID.

Consider code below as starting point:

https://github.com/OutpostUniverse/NetFixClient/blob/ff9e634120dfb3ce04e126d9d2470096557743d2/OPUNetTransportLayer.cpp#L1125-L1126

https://github.com/OutpostUniverse/NetFixClient/blob/ff9e634120dfb3ce04e126d9d2470096557743d2/OPUNetTransportLayer.cpp#L1406-L1411

### PR DESCRIPTION
**Pull request by Brett208**
Closes #56

### CODE REVIEW

File: OPUNetTransportLayer.cpp [hunk 1]
```diff
@@ -1114,7 +1114,7 @@ bool OPUNetTransportLayer::DoImmediateProcessing(Packet &packet, sockaddr_in &fr
 				tlMessage.tlHeader.commandType = tlcJoinGranted;
 				// **DEBUG**
 				Log("Client join accepted: " + FormatAddress(fromAddress) + 
-					" (" + std::to_string(tlMessage.joinReply.newPlayerNetID) + ")");
+					FormatPlayerNetID(tlMessage.joinReply.newPlayerNetID));
```
Comment (DanRStevens) [inline OPUNetTransportLayer.cpp:1117]: This might need some spacing between the address and the playerNetId.

**Review by DanRStevens — APPROVED** [review]
Looks good, though might have a small cosmetic issue.

File: OPUNetTransportLayer.cpp (hunk 1 above)
Comment (Brett208) [reply OPUNetTransportLayer.cpp:1117]: Tried to add some spacing. Please suggest how you want it to read if this doesn't look good. Better than me guessing what you are thinking.

**Comment by Brett208** [comment]
If this looks good, please merge.

Thanks,
Brett

File: OPUNetTransportLayer.cpp (hunk 1 above)
Comment (DanRStevens) [reply OPUNetTransportLayer.cpp:1117]: I noticed that the space between the IP address and the playerNetId had been removed. (Assuming I read the code correctly). This seems to resolve any potential reading difficulties.

I'm not about the test the output. If it looks good to you, go ahead.

Comment (Brett208) [reply OPUNetTransportLayer.cpp:1117]: Sounds good.

**Pull request merged**